### ISSUE TITLE
JAWS 17 + IE11 does not announce aria-describedby text for drop downs where selected option is inside optgroup

### ISSUE THREAD

**Issue by gburghardt** [FreedomScientific/VFO-standards-support#139]
### Summary

JAWS fails to read off the `aria-describedby` text for a drop down when tabbing into a drop down where the selected option is inside an `<optgroup>` tag.

### Expected result

JAWS announces the label and aria-describedby text.

### Actual result

The label was announced, but the text referenced by the aria-describedby attribute does not get announced.

### Example

```html
<!DOCTYPE html>
<html lang="en-us">
<head>
  <meta charset="utf-8">
  <title>Option group issue</title>
</head>
<body>
  <form method="get" action="#">
    <p id="a">Color</p>
    <p id="b">That is not a color</p>
    <ol>
      <li>
        <select aria-describedby="a b">
          <option value="">Select Foo</option>
          <optgroup label="Colors">
            <option value="grape">Grape</option>
            <option value="red">Red</option>
          </optgroup>
        </select>
      </li>
      <li>
        <select aria-describedby="a b">
          <option value="">Select Foo</option>
          <optgroup label="Colors">
            <option value="grape" selected>Grape</option>
            <option value="red">Red</option>
          </optgroup>
        </select>
      </li>
    </ol>
  </form>
</body>
</html>
```

1. Tab in to the first select box (JAWS announces the aria-describedby text)

2. Tab into the second select box

#### JAWS version and build number

17.0.2729
 
#### Operating System and version

Windows 10 Enterprise (Version 10.0.15063, Build 15063)

#### Browser and version:

Internet Explorer 11.1324.15063.0

**Comment by stevefaulkner** [comment]
test case: https://s.codepen.io/stevef/debug/VVQZeE

**Comment by stevefaulkner** [comment]
**Note:** tested with JAWS 2019.1801 - we only test with latest iteraction of JAWS as it is the verison that will get fixed.

@gburghardt try the third and forth test case I added ([codepen](https://codepen.io/stevef/pen/VVQZeE?editors=1000))  I don't think it is because its in an `optgroup`, i think its because the `option` has a `selected` attribute, which means the focus is moved to the option rather than the `select` container, this results in the `aria-describedby` not being announced. Note I added a case where the `aria-describedby` is on the `selected option` and it is announced.

` <option value="grape" selected aria-describedby="a b">Grape</option>`

**Comment by stevefaulkner** [comment]
**Note:** tested with JAWS 2019.1801 - we only test with latest iteraction of JAWS as it is the verison that will get fixed.
 Looking into this further, [@jnurthen mentioned on twitter](https://twitter.com/jnurthen/status/1065422787228053504) that the combobox received focus in all cases. Which is correct. What I found was that IE does not expose the `aria-describedby` or `aria-labelledby` with multiple references in the accessibility tree, values as it should. So no other AT report the describedby text for a `select` element in IE, when tabbing to the controls. JAWS has limited support, which it provides itself in order to bridge the gap in IE's accessibility support, but JAWS support itself has limitations(which you found). A possible workaround is to use aria-labelledby to concatenate the multiple text strings:

```<select id="s5" aria-label="select 5" aria-labelledby="s5 a b">
          <option value="">Select Foo</option>
          <optgroup label="Colors">
            <option value="grape" selected>Grape</option>
            <option value="red">Red</option>
          </optgroup>
        </select>
```
JAWS does announce the concaneted `aria-labelledby` for the cases where an `option` is `selected` and/or in an `optgroup`. 
Note when navigation to the select elements using the virtual cursor navigation key **C** in bothe JAWS and NVDA, the aria-describedby content is announced in all circumstances. 

Am marking this as wontfix as IE is legacy, the correct info is announced in virtual cursor mode, the bugs in IE won't be fixed and if needed to work for tabbing, the workaround can be used, although this is still not functioning in NVDA when tabbing.
